java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -254,7 +273,7 @@ protected static IntegerReader createIntegerReader(OrcProto.ColumnEncoding.Kind
       }
     }
 
-    public void startStripe(StripePlanner planner) throws IOException {
+    public void startStripe(StripePlanner planner, ReadLevel readLevel) throws IOException {
```

Comment:
Do we really need the readLevel for start stripe? is startStripe(planner, FOLLOW) where we plan/read the next set of data?

Reply:
Yes, that is how currently we trigger [startStripe for FOLLOW](https://github.com/apache/orc/pull/635/files#diff-8678415090c906d11e0aa6241ba24643192cd66617c68d8deb0f7efe74dd2ba3R1311)